README.md

```
@@ -757,6 +765,112 @@ You can interact with the `trash` similar to a standard directory, with some res
+  `Favorites` album filters, `sync-cursor`, `sync`, `watch`, and
+  favorite/unfavorite mutations
+- Legacy Shared Albums `shared`: old shared-stream adapter only, not a valid
+  CloudKit browse/sync target
+
+Typical browse and sync examples:
+
+```console
+$ icloud photos libraries --username jappleseed@apple.com
+$ icloud photos albums --username jappleseed@apple.com
+$ icloud photos list --album Screenshots --limit 20 --username jappleseed@apple.com
+$ icloud photos list --library shared:<zoneName> --limit 20 --username jappleseed@apple.com
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)